source/validate_instruction.cpp

```diff
@@ -67,6 +67,36 @@ spv_result_t CapabilityError(ValidationState_t& _, int which_operand,
          << " requires one of these capabilities: " << required_capabilities;
 }
 
+// Returns an opcode's required capabilities.
```

Comment:
It's not required, but rather enabling. Required means that all of them are required. But in reality any of the capabilities in the list can enable the opcode.

Reply:
Fixed